ci/new-instance.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -eu
+
+function log {
+   echo "$(date +"%T") - INFO - $*"
+}
+
+read -r -p "Cluster (test or production): " CLUSTER
+
+if [ -z "${CLUSTER}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+log "CLUSTER: $CLUSTER"
+
+read -r -p "lumen url:" URL
+
+if [ -z "${URL}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+read -r -p "lumen organisation title:" ORGANISATION
+
+if [ -z "${ORGANISATION}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+read -r -p "user email:" EMAIL
+
+if [ -z "${EMAIL}" ]; then
+    log "Nothing typed"
+    exit 2
+fi
+
+
+log "URL: $URL"
```

Comment:
Do you want to validate the url? I see in the doc that there are some requirements. 

Or maybe at least put that requirement in the "read URL" in line 18

Reply:
added requirement in the read URL step